lms/services/jstor/_article_metadata.py

```diff
@@ -15,6 +15,15 @@ class _ArticleMetadataSchema(RequestsResponseSchema):
     title = fields.Str()
     subtitle = fields.Str()
 
+    # Ids of the related items
+    next = fields.Str()
+    previous = fields.Str()
+
+    # Titles of the containing item
+    book_title = fields.Str()
```

Comment:
So the article has title but it might belong to a journal/book which itself has a title.

Reply:
That's right (as far as I know).

Reply:
Correct. JSTOR have two separate fields here (_either_ `book_title` or `journal` will be set).